pkg/image/archive.go

```diff
@@ -41,6 +43,11 @@ func (img *Archive) InstantiateManifest(m *manifest.Manifest,
 
 	tarPipeline := manifest.NewTar(buildPipeline, osPipeline, "archive")
 	tarPipeline.SetFilename(img.Filename)
+
+	tarPipeline.Compression = osbuild.TarArchiveCompression(img.Compression)
+	if err := tarPipeline.Compression.Validate(); err != nil {
+		return nil, err
+	}
```

Comment:
This isn't entirely necessary, is it?  It'll be caught by the stage creation later anyway.  Doesn't hurt though.